docs/user-guide/layer-settings.md

```diff
@@ -134,6 +139,110 @@ The editor is easy to approach thanks also to the following functions:
+
+### Style Methods 
+
+Different styles methods can be used for each symbolizer: by clicking on the <img src="../img/button/options_button.jpg" class="ms-docbutton"/> button, present on the panel of each symbolizer, the editor can choose between:
+
+* *Simple style*
+
+* *Classification style*
+
+* *Pattern mark style*
+
+* *Patter icon style*
+
+#### Simple style
+
+The Simple style is the default style described above for each symbolizer. 
+
+#### Classification style
+
+[MapStore](https://mapstore.geo-solutions.it/mapstore/#/) allows you to classify the style based on the attributes of the layer. The *Classification style* is available for *Marker*, *Line*, *Fill* and *Text* by clicking on the <img src="../img/button/options_button.jpg" class="ms-docbutton"/> button and choosing the <img src="../img/button/classification_style_button.jpg" class="ms-docbutton"/> options from the dropdown menu. 
+
+<img src="../img/layer-settings/classification_styl_panel.jpg" class="ms-docimage"  style="max-width:500px;">
+
+It this case the editor is allowed to choose a `Color ramp` and `Reverse order` of the color, the `Attribute` with which to classify the level, the `Method` (*Quantile*, *Equal interval*, *Natural breaks* and *Standard deviation*), the `Intervals` (px) and the `Opacity` (%). En example of the *Classification style* for a *Fill* can be the following one:
+
+<img src="../img/layer-settings/classification_style_ex.jpg" class="ms-docimage">
+
+#### Pattern mark style
+
+With the *Pattern mark style* you can edit the *Line* or the *Fill* and add a mark by clicking on the <img src="../img/button/options_button.jpg" class="ms-docbutton"/> button and choosing the <img src="../img/button/pattern_mark_style_button.jpg" class="ms-docbutton"/> options from the dropdown menu. 
+
+<img src="../img/layer-settings/classify_mark_panel.jpg" class="ms-docimage"  style="max-width:500px;">
+
+The editor can modify the *Mark* as explained [here](layer-settings.md#mark) and the basic [line](layer-settings.md#line) or [fill](layer-settings.md#fill) options. Take a look at the following example of the *Pattern mark style* for the *Line*.
+
+<img src="../img/layer-settings/classify_mark_ex.jpg" class="ms-docimage">
+
+#### Patter icon style
+
+With the *Pattern icon style* you can edit the *Line* or the *Fill* and add an icon by clicking on the <img src="../img/button/options_button.jpg" class="ms-docbutton"/> button and choosing the <img src="../img/button/pattern_icon_style_button.jpg" class="ms-docbutton"/> options from the dropdown menu. 
+
+<img src="../img/layer-settings/classify_icon_style_panel.jpg" class="ms-docimage"  style="max-width:500px;">
+
+The editor can modify the *Icon* as explained [here](layer-settings.md#icon) and the basic [line](layer-settings.md#line) or [fill](layer-settings.md#fill) options. Take a look at the following example of the *Pattern icon style* for the *Fill*.
```

Comment:
The style editor can configure the *Icon* as explained [here](layer-settings.md#mark) along with the usual options available for rules of type [Line](layer-settings.md#line) or [Fill](layer-settings.md#fill) depending on the selected symbolizer. Take a look at the following example of *Pattern icon style* for a *Fill* rule sample.